detekt-api/src/test/kotlin/io/gitlab/arturbosch/detekt/api/EntitySpec.kt

```diff
@@ -14,9 +14,10 @@ import java.nio.file.Paths
 class EntitySpec : Spek({
 
     describe("entity signatures") {
-
+        val path = Paths.get("/full/path/to/Test.kt")
         val code by memoized(CachingMode.SCOPE) {
-            compileContentForTest("""
+            compileContentForTest(
+                """
```

Comment:
minor formatting improvement

```suggestion
            compileContentForTest("""
```